src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -209,6 +209,10 @@ uct_rc_mlx5_iface_common_poll_rx(uct_rc_mlx5_iface_common_t *mlx5_common_iface,
 done:
     max_batch = rc_iface->super.config.rx_max_batch;
     if (rc_iface->rx.srq.available >= max_batch) {
+        if (rc_iface->rx.srq.reserved) {
```

Comment:
the `uct_rc_mlx5_iface_srq_post_recv` also called from `uct_rc_mlx5_iface_common_init`. Need to implement extra function parameter to distinguish the calls. I will do it if you like.